R_Exercises8.Rmd

```diff
@@ -21,30 +24,41 @@ place. Do **NOT** rename the file.
 
 Check your work by knitting the document.
 
-1. Create a level 2 header below in Markdown with the name "Answer 1".
+1. Create a level 2 header below in Markdown with the name
+"Answer 1".
 
+## Answer 1
 
-2. Add an unordered Markdown list of your favorite food dishes and
-add their ingredients as sub-items in the list. You should enter at least two
-dishes with two ingredients each.
+2. Add an unordered Markdown list of your favorite food dishes and add their ingredients as sub-items in the list. You should enter at least two dishes with two ingredients each.
```

Comment:
Changes to the original text make it harder to read the Pull Request. Only respond to the questions by adding the response.